src/coreclr/jit/codegencommon.cpp

```diff
@@ -11855,7 +11855,7 @@ void CodeGen::genMultiRegStoreToLocal(GenTreeLclVar* lclNode)
             {
                 if (!lclNode->AsLclVar()->IsLastUse(i))
                 {
-                    GetEmitter()->emitIns_S_R(ins_Store(srcType), emitTypeSize(srcType), reg, fieldLclNum, 0);
+                    GetEmitter()->emitIns_S_R(ins_Store(srcType), emitTypeSize(destType), reg, fieldLclNum, 0);
```

Comment:
is not it strange that `ins_Store` and `emitTypeSize` have different arguments here? 
Also, `ins_Store`  is usually taking destType, not source, it does not matter on x64 but I expect arm.

Reply:
Yeah, that’s how I got confused and did the wrong thing earlier.

in this case, ins_Store needs to take srcType because we might have a float variable in an int register. For example we hit this case on x86 when calling an unmanaged method that returns a struct of floats and we want to store the floats on the stack, which is what prompted my fix in the first place.